packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -15,86 +15,145 @@
+
+  const headingId = `bpk-bottom-sheet-heading-${id}`;
+  const dialogClassName = getClassName(
+    'bpk-bottom-sheet',
+    wide && 'bpk-bottom-sheet--wide'
+    );
+
+    return <BpkBreakpoint query={BREAKPOINTS.ABOVE_MOBILE}>
+      {(isAboveMobile: boolean) =>
+      <BpkDialogWrapper
+      ariaLabelledby={ariaLabelledby}
+      dialogClassName={dialogClassName}
+      id={id}
       isOpen={isOpen}
-      onClose={handleClose}
+      onClose={(
+        arg0?: TouchEvent | MouseEvent | KeyboardEvent | SyntheticEvent<HTMLDialogElement, Event>,
+        arg1?: {
+          source: 'ESCAPE' | 'DOCUMENT_CLICK';
+        }) => handleClose( isAboveMobile ? 0 : animationTimeout, arg0, arg1)}
+      exiting={exiting}
+      transitionClassNames={{
+        appear: getClassName('bpk-bottom-sheet--appear'),
+        appearActive: getClassName('bpk-bottom-sheet--appear-active'),
+        exit: getClassName('bpk-bottom-sheet--exit')
+      }}
       closeOnEscPressed={closeOnEscPressed}
-      renderTarget={renderTarget}
+      closeOnScrimClick={closeOnScrimClick}
+      timeout={{appear: animationTimeout, exit: isAboveMobile ? 0 : animationTimeout}}
       >
-      <ScrimBpkBottomSheetInner
-        id={id}
-        onClose={handleClose}
-        closeOnScrimClick={closeOnScrimClick}
-        containerClassName={getClassName('bpk-bottom-sheet--container')}
-        title={title}
-        closeLabel={closeLabel}
-        actionText={actionText}
-        onAction={onAction}
-        wide={wide}
-        exiting={exiting}
-        {...rest}
-      />
-    </Portal>
+      <>
+        <header className={getClassName('bpk-bottom-sheet--header')}>
+          <BpkNavigationBar
+            id={headingId}
+            className={getClassName('bpk-bottom-sheet--navigation')}
+            title={title &&
+              <h2
+                id={headingId}
+                className={getClassName('bpk-bottom-sheet--heading')}
+              >
+                {title}
+              </h2>
+            }
+            leadingButton={
+              <BpkCloseButton
+                className={getClassName('bpk-bottom-sheet--close-button')}
+                label={closeLabel}
+                onClick={(
+                  arg0?: TouchEvent | MouseEvent | KeyboardEvent | SyntheticEvent<HTMLDialogElement, Event>,
+                  arg1?: {
+                    source: 'ESCAPE' | 'DOCUMENT_CLICK';
+                  }) => handleClose( isAboveMobile ? 0 : animationTimeout, arg0, arg1)}
+              />
+            }
+            trailingButton={
+              actionText && onAction ? (
+                <BpkButtonLink
+                  className={getClassName('bpk-bottom-sheet--action-button')}
+                  onClick={onAction}
+                >
+                  {actionText}
+                </BpkButtonLink>
+              ) :
+              <div
+                className={getClassName('bpk-bottom-sheet--action-button')}
+              />
```

Comment:
Can we change this to
```suggestion
              null
```

As having this empty div causes some weird alignment behaviour when its just placed

![Screenshot 2024-03-05 at 12 08 14](https://github.com/Skyscanner/backpack/assets/8831547/a556d2b3-bdd7-4feb-8cc5-427b23ddd5eb)